pages/api/v1/answers.ts

```diff
@@ -585,7 +585,14 @@ async function Angela( // Good ol' Angela
         ) {
           const { url, requestOptions } =
             constructHttpRequest(actionToHttpRequest);
-          let out = await makeHttpRequest(url, requestOptions, currentHost);
+          let out;
+          try {
+            out = await makeHttpRequest(url, requestOptions, currentHost);
+          } catch (e) {
+            console.error(e);
+            // @ts-ignore
+            out = `Failed to call ${url}\n\n${e.toString()}`;
+          }
           out = processAPIoutput(out, chosenAction);
```

Comment:
doesn't this function expect an object and you've turned out into a string?

Reply:
_Very_ good point! :+1: 

Was confused by type of `out` being `any`